src/index.ts

```diff
@@ -1011,12 +1023,20 @@ export class ValidationService {
 
     private untrackFormInput(form: HTMLFormElement, inputUID: string) {
         let formUID = this.getElementUID(form);
-        if (!this.formInputs[formUID]) {
+        let formInputs = this.formInputs[formUID]
+        if (!formInputs) {
             return;
         }
         let indexToRemove = this.formInputs[formUID].indexOf(inputUID);
```

Comment:
Since the new code sets `formInputs` to `this.formInputs[formUID]`, this could be changed to:

```suggestion
        let indexToRemove = formInputs.indexOf(inputUID);
```